CHANGELOG.md

```diff
@@ -2,10 +2,31 @@
 
 This file summarizes notable changes introduced in aktualizr version. It roughly follows the guidelines from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is deemed necessary. Thus it does not exactly map to months of the year.
+Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is issued. Thus `N` does not necessarily map to months of the year.
 
 ## [??? (unreleased)]
 
+
+## [2020.1] - 2020-01-16
+
+### Changed
+
+- Increased Targets metadata file size limit: [PR](https://github.com/advancedtelematic/aktualizr/pull/1476)
+- Do not provision if the Primary times out while connecting to Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1491)
+- Check and fetch root metadata according to the Uptane standard: [PR](https://github.com/advancedtelematic/aktualizr/pull/1501)
```

Comment:
Aren't these more fixes (1501, 1510)?

Reply:
I debated that. I argue that nothing was broken before in either case. I think both items are definite improvements, but to me, "fix" implies that there was a bug.

Reply:
Ok, fair enough.